### PR TITLE
Enable Profiling tab for mysql (PDO) queries and disable cache in debug mode

### PR DESCRIPTION
**Pull request by csthomas**
Pull Request for Issue #13265.

### Summary of Changes

1. This PR makes the query cache disabled when db driver runs in debug mode.
2. Enable Profile for Mysql PDO.

Not direct forked but based on #13268. 

### Testing Instructions
Details at  #13268

1. Enable debug mode in global config and system debug plugin
2. Test at least one of driver: Mysql, MySqli or PDO Mysql. 
3. Check debug console "Database Queries" and "Profile" tab. Each query should run without using database cache. 

### Expected result
No cached query.
On PDOMYSQL driver profile tab is enabled.

### Actual result

On debug mode queries use database cache.
On PDOMYSQL driver profile tab is disabled.

### Documentation Changes Required
No

### CODE REVIEW

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on 8b23be55dbaf935c68982bce07aaac993be5fa15<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/20322">issues.joomla.org/tracker/joomla-cms/20322</a>.</sub>

**Comment by csthomas** [comment]
At now it may be hard to follow the test instruction because the cache is disabled by default in mysql 5.7.

You will only see the difference in PDOMYSQL driver, in which the Profile tab will be visible.

To enable the cache, you can add the following lines to the mysql configuration file.
In case of Ubuntu 18 is `/etc/mysql/mysql.conf.d/mysqld.cnf`
```
query_cache_type = 1
query_cache_size = 10M
query_cache_limit = 10M 
```

**Comment by csthomas** [comment]
This PR is designed to display more relevant information when the debug mode is enabled.

Maybe there is one more person to test any driver and do simple code review on the rest.

**Comment by SharkyKZ** [comment]
I have tested this item :white_check_mark: successfully on 8b23be55dbaf935c68982bce07aaac993be5fa15<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/20322">issues.joomla.org/tracker/joomla-cms/20322</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/20322">issues.joomla.org/tracker/joomla-cms/20322</a>.</sub>
